src/session_manager.cpp

```diff
@@ -390,7 +390,8 @@ void SessionManager::send_chronos_update(std::string& timer_id,
                                          const std::string& opaque_data,
                                          SAS::TrailId trail)
 {
-  std::vector<std::string> tags = {"CALL"};
+  std::map<std::string, uint32_t> tags; tags["CALL"] = 1;
```

Comment:
[The online C++ reference](http://en.cppreference.com/w/cpp/container/map/map) gives some examples of creating maps with different constructors.

You should consider using the technique in example 5.